WebRoot/js/zimbraMail/share/view/dialog/ZmFolderPropertyView.js

```diff
@@ -183,14 +183,16 @@ function(event) {
     }
 
 	if (organizer.isSystem() || organizer.isDataSource()) {
-		this._nameOutputEl.innerHTML = AjxStringUtil.htmlEncode(organizer.name);
```

Comment:
we used to have a span previously . Not we just have a input field which is disabled/enabled.